ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot

```diff
@@ -117,25 +117,6 @@ Clean Up DSP Page
         Wait Until Page Contains    Data Science Projects
     END
 
-Try Opening Create Server
```

Comment:
don't you need to delete also `Clean Up DSP Page`?

Reply:
Right!